_im-plugin/ism/security.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: ISM security
+nav_order: 10
+parent: Index State Management
+has_children: false
+---
+
+# ISM security
+
+Using the security plugin with index state management lets you limit non-admin users to certain actions. For example, you might want to set up your security such that a group of users can only read ISM policies, while others can create, delete, or change policies.
+
+All index state management data are protected as system indices, and only a super admin or an admin with a Transport Layer Security (TLS) certificate can access system indices. For more information, see [System indices]({{site.url}}{{site.baseurl}}/security-plugin/configuration/system-indices).
+
+## Basic permissions
+
+The security plugin comes with two built-in roles that cover most ISM use cases: `ism_read_access` and `ism_full_access`. For descriptions of each, see [Predefined roles]({{site.url}}{{site.baseurl}}/security-plugin/access-control/users-roles#predefined-roles).
+
+With security enabled, users not only need the correct index management permissions, but they also need permissions to relevant indices to execute [REST API]({{site.url}}{{site.baseurl}}/im-plugin/ism/api) calls. For example, if a user wants to use the REST API to attach a policy to an index named `system-logs`, they would need the necessary REST API permissions as well as access to `system-logs`.
```

Comment:
This is true for all ISM apis except
* Create policy
* Get policy
* Delete policy

Wanted to call out in case you wanted to be more explicit here. Simply put any ISM API that interacts with managed index will require additional permission that user is allowed to manage the index i.e have the permission - `indices:admin/opensearch/ism/managedindex` on index